swig/comedi.i

```diff
@@ -115,14 +118,19 @@ unsigned int NI_AO_SCAN_BEGIN_SRC_RTSI(unsigned int rtsi_channel);
 %array_class(lsampl_t, lsampl_array);
 %array_class(comedi_insn, insn_array);
 
+#ifndef SWIGPYTHONONLYSHORT
 %insert("python") %{
 # Add entries in module dictionary to strip comedi_/COMEDI_ prefix
+import copy
 import re
-for k,v in globals().items():
+myglobals = copy.copy(globals())
+# Copy dictionary before 
+for k,v in myglobals.items():
   if re.match('^comedi_', k, flags=re.IGNORECASE):
     globals()[k[7:]] = v
     # uncommenting following line removes compatibility with old code using
```

Comment:
You could remove this comment about removing compatibility, as now there is a way to do so using `#define SWIGPYTHONONLYSHORT`.